packages/web-app-media-viewer/src/App.vue

```diff
@@ -167,7 +167,18 @@ export default {
         a: 1
       }
 
-      return this.$_loader_getDavFilePath(this.activeMediaFile.path, query)
+      if (this.$_loader_publicContext && this.activeMediaFile.downloadURL) {
```

Comment:
Could you move this entire block into the `$_loader_getDavFilePath` mixin and just provide the entire `activeMediaFile` instead of the path? Then you're also able to get rid of the path traversal with `../../../`. The URL format is well known to us, I'd prefer string replaces instead of traversals if possible.